jobs/build-snaps/build.sh

```diff
@@ -28,16 +28,16 @@ git clone https://github.com/juju-solutions/release.git --branch rye/snaps --dep
 )
 
 rm -rf ./cdk-addons
-if git ls-remote --exit-code --heads https://github.com/juju-solutions/cdk-addons.git ${ADDONS_BRANCH_VERSION}
+if git ls-remote --exit-code --heads https://github.com/charmed-kubernetes/cdk-addons.git ${ADDONS_BRANCH_VERSION}
 then
   echo "Getting cdk-addons from ${ADDONS_BRANCH_VERSION} branch."
-  git clone https://github.com/juju-solutions/cdk-addons.git --branch ${ADDONS_BRANCH_VERSION} --depth 1
+  git clone https://github.com/charmed-kubernetes/cdk-addons.git --branch ${ADDONS_BRANCH_VERSION} --depth 1
   if [ "$SKIP_RELEASE_TAG" != "true" ]; then
     tag_release  "juju-solutions" "cdk-addons" ${ADDONS_BRANCH_VERSION} ${GH_USER} ${GH_TOKEN} ${KUBE_VERSION}
```

Comment:
@battlemidget does `juju-solutions` need to be changed to `charmed-kubernetes` throughout this file as well?

Reply:
Yea i think so, cdkbot is part of charmed kubernetes so we should be good there